src/modules/test/main.c

```diff
@@ -365,7 +376,8 @@ int RunCommand(const COMMAND_STEP* command)
 int RunTestStep(const TEST_STEP* test, const MANAGEMENT_MODULE* module)
 {
     int result = 0;
-    JSON_Value* value = NULL;
+    JSON_Value* actual = NULL;
+    JSON_Value* expected = NULL;
```

Comment:
nit: better to say what is expected or actual in the names. 'expectedPayload', for example, is clearer than 'expected'